Make.hs

```diff
@@ -444,7 +444,6 @@ execute command args = do
 
 -- | For parallel RGL module compilation
 -- Unfortunately, this has no effect unless compiled with -threaded
-parallel_ :: (Foldable t, Monad m) => t (m a) -> m ()
```

Comment:
Could you comment this instead of deleting the line?

Reply:
there you go!

Reply:
I rebased the commit and force-pushed, so the github interface looks kind of strange